README.md

```diff
@@ -0,0 +1,24 @@
+# ros2cli
+
+This repository contains the source code for ROS 2 command line interface tools included with a standard install of any ROS 2 distro. 
+
+## Usage
+
+Run `ros2 --help` to see all available verbs.
+
+Run `ros2 <verb> --help` for more information on individual command usage.
+
+Read [Introspection with command line tools](https://index.ros.org/doc/ros2/Tutorials/Introspection-with-command-line-tools/) on ROS Index for more information and an example.
+
+### Cheat Sheet
+
+This [cheat sheet](https://github.com/artivis/ros2_cheats_sheet/blob/master/cli/cli_cheats_sheet.pdf) provides examples for verbs and their sub-verbs (some of which rely on the [ROS 2 demos package](https://github.com/ros2/demos)).
+
+## Add New Verbs
+
+You can use [Python entry points](https://setuptools.readthedocs.io/en/latest/pkg_resources.html#entry-points) to create new verbs and sub-verbs for the cli. 
+[Here's an example](https://github.com/ros2/ros2cli/pull/273/files).
```

Comment:
We would need also an example on how to add a command. It's not exactly the same.
Here is an [example](https://github.com/ros2/ros2cli/pull/288/files), but it's too long.

Reply:
I guess we could put together a `ros2 hello world`

Reply:
That would be great!

Reply:
FYI, wip [ros2hellocli](https://github.com/artivis/ros2hellocli).